akka-docs/src/main/paradox/stream/operators/Sink/foreachAsync.md

```diff
@@ -0,0 +1,33 @@
+# foreachAsync
+
+Invoke a given procedure asynchronously for each element received.
+
+@ref[Sink stages](../index.md#sink-stages)
+
+@@@div { .group-scala }
```

Comment:
no java signature?

Reply:
there is no "@scala-signature" nor "@java-signature", just "@signature" which translates to Scala code, so, I can't see the point to add it. is there any other way to add the Java signature?